plugins/uefi/fu-plugin-uefi.c

```diff
@@ -58,6 +58,9 @@ fu_plugin_init (FuPlugin *plugin)
 	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_RUN_AFTER, "upower");
 	fu_plugin_add_report_metadata (plugin, "FwupdateVersion", LIBFWUP_LIBRARY_VERSION);
 	fu_plugin_add_report_metadata (plugin, "EfivarVersion", EFIVAR_LIBRARY_VERSION);
+#if LIBFWUP_CHECK_VERSION(12,0)
+	fu_plugin_add_runtime_version (plugin, "com.redhat.fwupdate", fwup_version ());
+#endif
```

Comment:
As a fallback for earlier versions of `fwupdate` on the system how about rely on the compile time version?  Your call if that makes sense.

Reply:
I'm explicitly marking this the runtime version; i think it would be confusing to fall back especially when the runtime version could be less than the build-time version on a deployed system.

Reply:
OK.